api/src/main/java/com/codeforcommunity/rest/ApiRouter.java

```diff
@@ -54,6 +59,7 @@ public Router initializeRouter(Vertx vertx) {
 
     router.mountSubRouter("/user", authRouter.initializeRouter(vertx));
     router.mountSubRouter("/webhooks", webhooksRouter.initializeRouter(vertx));
+    router.mountSubRouter("/events_v2", publicEventsRouter.initializeRouter(vertx));
```

Comment:
```suggestion
    router.mountSubRouter("/events", publicEventsRouter.initializeRouter(vertx));
```
I think `api/v1/protected/events` and `api/v1/events` is enough of a distinction, and this new route isn't necessarily a full V2, it's just different concerns for what data it returns. also `api/v1/events_v2` looks a bit odd.